h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -114,6 +116,7 @@
     private boolean enableGLRMReconstrut = false;
     private boolean enableStagedProbabilities = false;
     private boolean enableContributions = false;
+    private int glrmIterNumber = 100; // default set to 1009
```

Comment:
typo? comment in contrast whith reality commenting something that is apparent

Reply:
Good catch.  Fat fingers.  Fixed.